packages/edit-site/src/components/page-pages/index.js

```diff
@@ -31,6 +32,12 @@ export default function PagePages() {
 			field: 'date',
 			direction: 'desc',
 		},
+		fields: {
+			// All fields are visible by default, so it's
+			// better to keep track of the hidden ones.
```

Comment:
I'm not sure this assumption is true for all the views. I'd think that it's probably the opposite: most fields should be hidden by default but I'm ok starting this way.